src/main/java/com/pss/interfaces/IGetProjectileDrag.java

```diff
@@ -3,5 +3,6 @@
 import javax.vecmath.Vector3d;
 
 public interface IGetProjectileDrag {
-    public Vector3d getProjectileDrag();
+    // public Vector3d getProjectileDrag();
+    public double getProjectileDrag();
```

Comment:
Same comment as before. I think we should keep the interface returning a Vector. Making it return a number I think is more work, that we don't need to do